### PR TITLE
add helm chart of csi-qingcloud

### PR DESCRIPTION
**Pull request by min-zh**
fix readme of csi-neonsan by the way.

### CODE REVIEW

File: src/test/csi-qingcloud/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+apiVersion: v2
+
+appVersion: 1.2.0
+name: csi-qingcloud
+description: A Helm chart for Qingcloud CSI Driver
+version: 1.2.0-rc1
```
Comment (hlwanghl) [inline src/test/csi-qingcloud/Chart.yaml:19]: Can we use a formal version without `-rc1`? Helm v3 ignores pre-releases by default.

File: src/test/csi-qingcloud/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+kubeVersion: ">=1.14.0-0"
+home: https://github.com/yunify/qingcloud-csi
+sources:
+  - https://github.com/yunify/qingcloud-csi
+keywords:
+  - qingcloud
+  - csi
+maintainers:
+  - name: min-zh
```
Comment (hlwanghl) [inline src/test/csi-qingcloud/Chart.yaml:28]: Including an email address is better.

File: src/test/csi-qingcloud/templates/node-ds.yaml [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+            - "--csi-address=$(ADDRESS)"
+            - "--kubelet-registration-path=/var/lib/kubelet/plugins/{{ .Values.driver.name }}/csi.sock"
+            - "--v=5"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sh", "-c", "rm -rf /registration/disk.csi.qingcloud.com /registration/{{ .Values.driver.name }}-reg.sock"]
```
Comment (hlwanghl) [inline src/test/csi-qingcloud/templates/node-ds.yaml:72]: Should this use the variable `driver.name` in `"rm -rf /registration/{{ .Values.driver.name }}"`?

File: src/test/csi-qingcloud/templates/node-ds.yaml [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+            - name: server-config
+              mountPath: /etc/config
+      volumes:
+        - name: socket-dir
+          hostPath:
+            path: "/var/lib/kubelet/plugins/{{ .Values.driver.name }}/"
+            type: DirectoryOrCreate
+        - name: registration-dir
+          hostPath:
+            path: /var/lib/kubelet/plugins_registry/
```
Comment (hlwanghl) [inline src/test/csi-qingcloud/templates/node-ds.yaml:125]: Can we make kubelet path customizable so QKE can use `/data/var/lib/kubelet`?

File: src/test/csi-qingcloud/templates/node-ds.yaml (hunk 3 above)
Comment (min-zh) [reply src/test/csi-qingcloud/templates/node-ds.yaml:72]: yes，i have missed this one

File: src/test/csi-qingcloud/Chart.yaml (hunk 2 above)
Comment (min-zh) [reply src/test/csi-qingcloud/Chart.yaml:28]: OK,

File: src/test/csi-qingcloud/Chart.yaml (hunk 1 above)
Comment (min-zh) [reply src/test/csi-qingcloud/Chart.yaml:19]: ok

File: src/test/csi-neonsan/Chart.yaml [hunk 5]
```diff
@@ -25,5 +25,5 @@ keywords:
   - neonsan
   - csi
 maintainers:
-  - name: min-zh
+  - name: arminzhang@yunify.com
```
Comment (hlwanghl) [inline src/test/csi-neonsan/Chart.yaml:28]:
```
maintainers:
- name: Min Zhang
  email: xxx@xx.com
```

Comment (min-zh) [reply src/test/csi-neonsan/Chart.yaml:28]: OK

**Pull request merged**